package.json

```diff
@@ -33,7 +36,10 @@
     "jsonwebtoken": "^8.4.0"
   },
   "peerDependencies": {
-    "middy": "^0.22.0"
+    "@middy/core": "^1.0.0",
+    "@middy/http-error-handler": "^1.0.0",
+    "@middy/http-header-normalizer": "^1.0.0",
+    "@types/aws-lambda": "^8.10.51"
```

Comment:
I don't think these (except for @middy/core) should be peerDependencies as they are not required for using the package, only for testing it. Please move to devDependencies instead.

Reply:
They shouldn't be in dependencies either.